docs/tutorial/quick-start.md

```diff
@@ -0,0 +1,38 @@
+# # Quick start
```

Comment:
I advise choosing a different title than "Quick start". I suggest: 

- Deploy the GitHub runner image builder for the first time
- Deploy the `github-runner-image-builder-operator` charm for the first time

Let me know what you think!

Reply:
Sounds good! Updated :)